data/translations/en/state-data.json

```diff
@@ -341,13 +380,16 @@
     "more_info_link": "https://elect.ky.gov/Resources/Pages/Registration.aspx",
     "more_info_link_translated": "true",
     "more_info_link_language_selection": "",
-    "ip_deadline": "Fourth Monday before Election Day",
-    "online_deadline": "Fourth Monday before Election Day",
-    "mail_received_deadline": "Must be received by the fourth Monday before Election Day",
-    "mail_postmarked_deadline": "",
+    "online_deadline": "Tuesday, October 11, 2022",
+    "ip_deadline": "Tuesday, October 11, 2022",
+    "mail_postmarked_deadline": "Must be postmarked by Tuesday, October 11, 2022",
```

Comment:
Changed from received from postmarked. Which one should we use?

Reply:
confirmed